.github/workflows/build.yml

```diff
@@ -150,12 +150,12 @@ jobs:
         herbstluftwm &
         sleep 1
         python -c "import sys; print(sys.path)"
-        pytest -sv
+        pytest tests/ert_tests -sv
```

Comment:
just wondering, is there any reason to have it in `tests/ert_tests/...` instead of `tests/ert/...` (the same for libres) ?

Reply:
We actually started with this solution, but I found it a bit ambiguous. My terminal does not show me the full path, only the folder name I am in. So from one glance it is impossible to say whether you are in ert root or in ert test directory. Do you think it would not be a problem for many people?